components/sandbox/toolchain-host-operator/staging/stone-stage-p01/kustomization.yaml

```diff
@@ -5,3 +5,4 @@ resources:
 - toolchainconfig.yaml
 - space-provisioner-configs.yaml
 - network-policy.yaml
+- ../../../tiers/staging
```

Comment:
one question, what will cause host operator from using those tiers instead of the ones packaged in the operator?

Reply:
bacause the top NSTemplateTier resources will point to the new templates. For example right now appstudio tier points to these templates in staging:

```
apiVersion: toolchain.dev.openshift.com/v1alpha1
kind: NSTemplateTier
metadata:
  annotations:
    toolchain.dev.openshift.com/last-applied-configuration: '{"kind":"NSTemplateTier","apiVersion":"toolchain.dev.openshift.com/v1alpha1","metadata":{"name":"appstudio","namespace":"toolchain-host-operator","creationTimestamp":null},"spec":{"namespaces":[{"templateRef":"appstudio-tenant-eaadfb5-eaadfb5"}],"clusterResources":{"templateRef":"appstudio-clusterresources-64191fe-64191fe"},"spaceRoles":{"admin":{"templateRef":"appstudio-admin-42c5028-42c5028"},"contributor":{"templateRef":"appstudio-contributor-97cedbf-97cedbf"},"maintainer":{"templateRef":"appstudio-maintainer-97cedbf-97cedbf"},"viewer":{"templateRef":"appstudio-viewer-662ad5d-662ad5d"}}},"status":{}}'
  creationTimestamp: "2023-02-27T08:59:58Z"
  generation: 45
  name: appstudio
  namespace: toolchain-host-operator
  resourceVersion: "510819366"
  uid: 13fcfee9-6ecd-4807-9029-a958ef43abf8
spec:
  clusterResources:
    templateRef: appstudio-clusterresources-64191fe-64191fe
  namespaces:
  - templateRef: appstudio-tenant-eaadfb5-eaadfb5
  spaceRoles:
    admin:
      templateRef: appstudio-admin-42c5028-42c5028
    contributor:
      templateRef: appstudio-contributor-97cedbf-97cedbf
    maintainer:
      templateRef: appstudio-maintainer-97cedbf-97cedbf
    viewer:
      templateRef: appstudio-viewer-662ad5d-662ad5d

```
But once this PR merges it will use this one:

```
# ----------------------------------------------------------------
# Generated by cli - DO NOT EDIT
# ----------------------------------------------------------------

apiVersion: toolchain.dev.openshift.com/v1alpha1
kind: NSTemplateTier
metadata:
  name: appstudio
  namespace: toolchain-host-operator
spec:
  clusterResources:
    templateRef: appstudio-clusterresources-750679343-750679343
  namespaces:
  - templateRef: appstudio-tenant-1252037388-1252037388
  spaceRoles:
    admin:
      templateRef: appstudio-admin-221167138-221167138
    contributor:
      templateRef: appstudio-contributor-3696782055-3696782055
    maintainer:
      templateRef: appstudio-maintainer-98574726-98574726
    viewer:
      templateRef: appstudio-viewer-210715919-210715919
```

Reply:
Thank you for the explanation